### PR TITLE
Queryable interfaces

### PR DESCRIPTION
**Pull request by eldargab**
Example usage

```gql
interface Entity @query {
  id: ID!
  name: String
}

type Foo implements Entity @entity {
  id: ID!
  name: String
  foo: Int
}

type Bar implements Entity @entity {
  id: ID!
  name: String
  bar: Int
}

query {
  entities {
    id
    name
    ... on Foo {
      foo
    }
    ... on Bar {
      bar
    }
  }
}
```

### CODE REVIEW

**Comment by belopash** [comment]
Is there something like "type" to identify what entity it is?

**Comment by eldargab** [comment]
Standard `__typename` works. To sort by entity type use `orderBy: _type_ASC`

**Review by dzhelezov — APPROVED** [review]
LGTM (but see my comment for filtering on `_type`)

File: openreader/src/test/queryable.test.ts [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+                        id: '2',
+                        name: 'ref-2'
+                    },
+                    foo: 2
+                }
+            ]
+        })
+    })
+
+    it('sorting by entity type', function() {
```
[the remark this answers is not in the document]
Comment (eldargab) [reply openreader/src/test/queryable.test.ts:150]: This is not implemented.

**Pull request merged**